jormungandr/src/rest/v0/handlers.rs

```diff
@@ -255,3 +255,16 @@ pub fn get_shutdown(context: State<Context>) -> Result<impl Responder, Error> {
         .stop();
     Ok(HttpResponse::Ok().finish())
 }
+
+pub fn get_stake_pools(context: State<Context>) -> Result<impl Responder, Error> {
+    let blockchain = context.blockchain.lock_read();
+    let stake_pool_ids = blockchain
+        .tip
+        .ledger()
```

Comment:
I realize it was not made clear in the original issue, but I think this is was not the intended source of information. I think we want to list the stake pool at the moment it has been snapshotted for the current leadership, not the current stake pools defined in the tip ledger.

that being said, we can fix it later and accept this as a putting most of the REST call in place. Or we can also offer both the current tip and the snapshot (possibly with a different rest endpoint)

Reply:
Are both versions going to be usable?